src/hooks/useHTMLTextDirection.tsx

```diff
@@ -15,4 +15,18 @@ const useHTMLTextDirection = (direction: HTMLTextDirection) => {
   }, [direction]);
 };
 
+export const useMessageLayoutDirection = (direction: HTMLTextDirection, forceLeftToRightMessageLayout: boolean, loading: boolean) => {
+  useEffect(() => {
+    if (loading) return;
+    const messageListElements = document.getElementsByClassName('sendbird-conversation__messages');
```

Comment:
I think it's good to cover this scope with `try catch`

Reply:
Oh if there's no element to target, this will return an empty array if you're worry about this :)